src/framework/core/js/Fluid.js

```diff
@@ -1741,6 +1744,24 @@ var fluid = fluid || fluid_3_0_0;
         return fluid.makeArray(target).concat(fluid.makeArray(source));
     };
 
+    /*** FLUID LOGGING SYSTEM ***/
+
+    // This event represents the process of resolving the action of a request to fluid.log. Each listener shares
+    // access to an array, shallow-copied from the original arguments list to fluid.log, which is assumed writeable
+    // and which they may splice, transform, etc. before it is dispatched to the listener with namespace "log" which
+    // actually performs the logging action
+    fluid.loggingEvent = fluid.makeEventFirer({name: "logging event"});
+
+    fluid.addTimestampArg = function (args) {
+        var arg0 = fluid.renderTimestamp(new Date()) + ":  ";
+        args.unshift(arg0);
+    };
+
+    fluid.loggingEvent.addListener(fluid.doBrowserLog, "log");
+    // A placeholder for any actions filtering the log arguments before dispatching
+    fluid.loggingEvent.addListener(fluid.identity, "filterArgs", "before:log");
```

Comment:
I assume that you're meant to replace this for the centralized logging. I think it would probably be useful to have a test to verify that it works and to demonstrate how it would be replaced.

Reply:
No, the filterArgs just represents a position with the chain so that one can write after:filterArgs rather than writing before:log which might be more ambiguous. I'll improve the comment wording